tests/integration/test_trimmed_geometry.py

```diff
@@ -0,0 +1,229 @@
+# Copyright (C) 2023 - 2024 ANSYS, Inc. and/or its affiliates.
+# SPDX-License-Identifier: MIT
+#
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in all
+# copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+
+from ansys.api.geometry.v0.commands_pb2 import CreateSketchLineRequest
+import numpy as np
+import pytest
+
+from ansys.geometry.core.connection import BackendType, point3d_to_grpc_point
+from ansys.geometry.core.designer.design import Design
+from ansys.geometry.core.designer.face import SurfaceType
+from ansys.geometry.core.geometry import Circle, Line
+from ansys.geometry.core.geometry.box_uv import LocationUV
+from ansys.geometry.core.geometry.curves.trimmed_curve import ReversedTrimmedCurve, TrimmedCurve
+from ansys.geometry.core.geometry.parameterization import Interval
+from ansys.geometry.core.geometry.surfaces.trimmed_surface import (
+    ReversedTrimmedSurface,
+    TrimmedSurface,
+)
+from ansys.geometry.core.math import Point3D, UnitVector3D
+from ansys.geometry.core.math.point import Point2D
+from ansys.geometry.core.modeler import Modeler
+from ansys.geometry.core.sketch.sketch import Sketch
+
+
+def create_sketch_line(design: Design, p1: Point3D, p2: Point3D):
+    point1 = point3d_to_grpc_point(p1)
+    point2 = point3d_to_grpc_point(p2)
+    design._commands_stub.CreateSketchLine(CreateSketchLineRequest(point1=point1, point2=point2))
+
+
+def create_hedgehog(modeler: Modeler):
```

Comment:
Let's add some descriptions to these functions if possible as well as the tests. Use docstrings for this